bin/key_server.js

```diff
@@ -74,10 +74,10 @@ function main() {
     log.stat(Password.stat())
   }
 
-  require('../lib/mailer')(config, log)
+  require('../lib/senders')(config, log)
     .done(
-      function(m) {
-        mailer = m
+      function(result) {
+        senders = result
```

Comment:
`senders` is also referenced by the `shutdown` function further down, which is outside this scope.